tdp_vars_defaults/hbase/hbase.yml

```
@@ -3,11 +3,11 @@
 
 ---
 # HBase version
-hbase_release: hbase-2.1.10-TDP-0.1.0-SNAPSHOT
+hbase_release: hbase-2.1.10-1.0
 hbase_dist_file: "{{ hbase_release }}-bin.tar.gz"
 
 # Hbck2 version
-hbase_hbck2_release: hbase-hbck2-1.1.0-TDP-0.1.0-SNAPSHOT 
+hbase_hbck2_release: hbase-hbck2-1.1.0-0.0.jar 
```

Do not put `.jar` here.
```suggestion
hbase_hbck2_release: hbase-hbck2-1.1.0-0.0
```

---

Done in latest push